forui/CHANGELOG.md

```diff
@@ -327,10 +327,14 @@ The `hovered` state has also been split into `hovered` and `pressed` states.
 * Fix `FTappable`'s animation sometimes being invoked after it is unmounted.
 
 
-### `FTextField`
+### `FTextField` & `FTextFormField` (new)
```

Comment:
I think we should mention that this is a breaking change. And users should replace existing `FTextField` with `FTextFormField` to retain the same behavior.